packages/matter-node-ble.js/src/ble/BluetoothNode.ts

```diff
@@ -0,0 +1,37 @@
+/**
```

Comment:
I'd name this file `BleNode.ts`

Reply:
did, also renamed all rest from Bluetooth to ble